gradle.properties

```diff
@@ -5,7 +5,7 @@ cordaProductVersion = 5.2.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 30
+cordaApiRevision = JR
```

Comment:
```suggestion
cordaApiRevision = 31
```

Reply:
Yep, I'm waiting for reviews on both `PR`s before acquiring the lock and using the proper version. Thanks for pointing out anyways 👍